docs/getting_started.rst

```diff
@@ -1,9 +1,75 @@
 Getting Started
 ===============
+Installing sparcle_qc
+---------------------
+To get started, you will need to install sparcle_qc and its dependencies. We have provided a yaml file to create a conda environment containing all of the necessary packages.
```

Comment:
Do we want to use sparcle_qc or Sparcle_QC when talking about it outside of code?

Reply:
great point, I don't particularly mind either way, but maybe since our logo is Sparcle_QC, we should stick with that?

Reply:
yes, I like that!

Reply:
Updated